src/api/database/models/map/Landmark.ts

```diff
@@ -0,0 +1,37 @@
+import {
+  Column,
+  Table,
+  Model,
+  PrimaryKey,
+  ForeignKey,
+  Unique,
+  BelongsTo,
+} from "sequelize-typescript";
+import { UUID, STRING, JSONB, DATE } from "sequelize";
+import User from "../User";
+import ZodColumn from "../../modelHelpers/ZodColumn";
+import z from "zod";
+
+@Table
+class Landmark extends Model {
+  @PrimaryKey
+  @Column(STRING)
+  landmark: string;
+
+  @Unique
+  @PrimaryKey
+  @ForeignKey(() => User)
+  @Column(UUID)
+  userId: string;
+
+  @ZodColumn(JSONB, z.record(z.string(), z.any()).nullable())
+  evalutaion: Record<string, any> | null;
```

Comment:
split into two fields:

`score: number` with Zod validation of range [1, MAX_LANDMARK_SCORE]. Define MAX_LANDMARK_SCORE in shared/Map.ts and replace the `4` in that file with this constant.

`markdown: string`. This is a note field where the user can jog down notes when scoring. The mock UI design doesn't have this field but we will need to add a text input box in the real UI.